### ISSUE TITLE
Selecting a menu item with cursor and clicking F1 raises exception

### ISSUE THREAD

**Issue by delphidabbler** [delphidabbler/codesnip#54]
Error reported when running outside debugger is:

```
No context-sensitive help installed.
```

And when running under debugger:

```
Project CodeSnip.exe raised exception class EHelpSystemException with message 'No context-sensitive help installed'.
```

**Comment by delphidabbler** [comment]
This bug also presented itself in my _VIEd_ program.

The bug was fixed by adding a _TApplicationEvents_ component (named _ApplicationEvents_) and handling its _OnHelp_ event, using the following code:

```pascal
function TMainForm.ApplicationEventsHelp(Command: Word; Data: Integer;
  var CallHelp: Boolean): Boolean;
begin
  // Prevent Delphi Help system from interfering!
  // This prevents exception being raised when F1 is pressed over menu items
  // while still allowing our custom help manager to operate.
  CallHelp := False;
end;
```

This prevented the exception reported here while still allowing my custom HTML help system to be used.

**Comment by delphidabbler** [comment]
2ndary bug introduced in _VIEd_ using code above: function result missing from _TMainForm.ApplicationEventsHelp_ method. Code should be:

```pascal
function TMainForm.ApplicationEventsHelp(Command: Word; Data: Integer;
  var CallHelp: Boolean): Boolean;
begin
  // Prevent Delphi Help system from interfering!
  // This prevents exception being raised when F1 is pressed over menu items
  // while still allowing our custom help manager to operate.
  CallHelp := False;
  Result := True;
end;
```

**Comment by delphidabbler** [comment]
Fixed at commit a6fd189
